modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -322,7 +323,7 @@ public PartitionReplicaLifecycleManager(
         this.partitionOperationsExecutor = partitionOperationsExecutor;
         this.clockService = clockService;
         this.executorInclinedSchemaSyncService = new ExecutorInclinedSchemaSyncService(schemaSyncService, partitionOperationsExecutor);
-        this.placementDriver = placementDriver;
+        this.placementDriver = new ExecutorInclinedPlacementDriver(placementDriver, partitionOperationsExecutor);
```

Comment:
```suggestion
        this.executorInclinedPlacementDriver = new ExecutorInclinedPlacementDriver(placementDriver, partitionOperationsExecutor);
```
We use this naming pattern in `TableManager` and at least for schema sync service in `PartitionReplicaLifecycleManager`

Reply:
Renamed.